### ISSUE TITLE
I cant run the weblaf-demo-1.2.13-jar-with-dependencies.jar

### ISSUE THREAD

**Issue by flightcrank** [mgarin/weblaf#714]
I get an error when i execute the jar file with 

`jav -jar weblaf-demo-1.2.13-jar-with-dependencies.jar`

here is the error that is output

`ERROR com.alee.utils.CoreSwingUtils - Uncaught EventQueue exception: java.lang.ExceptionInInitializerError
java.lang.ExceptionInInitializerError`

strange because i have run this demo before on another system and it worked. but now its wont. Im on java 21

**Comment by jeddors** [comment]
Same for me.

Windows 11
java 23.0.1 2024-10-15
Java(TM) SE Runtime Environment (build 23.0.1+11-39)
Java HotSpot(TM) 64-Bit Server VM (build 23.0.1+11-39, mixed mode, sharing)

**Comment by mochawoof** [comment]
It works on older versions of Java, tested on Java 8

**Comment by flightcrank** [comment]
Yeah i got it working on java 8. the refelction api is used in this demo and its cause majot problems on a modern Java install. 
@mgarin are you ever going to fix this ?
